### PR TITLE
[MU4] Fix #12587: Images cannot be moved via drag or keyboard

### LINKED ISSUE #12587: [MU4 Issue] Images cannot be moved via drag or keyboard
**Describe the bug**
Cursor keys should nudge elements, and do for most (sometimes requiring edit mode), but for images, no nudging seems possibility.  This is a regression from MU3.

**To Reproduce**
1. In an empty, paste an image (that you capture to your clipboard elsewhere) onto the first rest
2. Try to move it via keyboard or mouse
3. See error - the image resizes rather than moves

**Expected behavior**
Image should resize if you drag it explicitly, or select a handle to adjust by keybaord, but default operation of both drag and cursor keys should be nudge.  Or, if a conscious choice is being made that resizing should be the default, then there should be an obvious way to move instead.  But edit mode doesn't work, nor does deselecting then dragging.

**Platform information**
- Linux

**Additional context**
I was able to drag by right-clicking - actually, to be more accurate, by dragging with two fingers on my touchpad.  This is entirely non-obvious and probably a bug in itself.

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/12587
Resolves: https://github.com/musescore/MuseScore/issues/11900

### CODE REVIEW

File: src/notation/internal/notationactioncontroller.cpp [hunk 1]
```diff
@@ -1244,6 +1244,8 @@ void NotationActionController::startEditSelectedElement(const ActionData& args)
     if (interaction->textEditingAllowed(element)) {
         PointF cursorPos = !args.empty() ? args.arg<PointF>(0) : PointF();
         interaction->startEditText(element, cursorPos);
+    } else if (element->hasGrips()) {
```
Comment (wizofaus) [inline src/notation/internal/notationactioncontroller.cpp:1247]: but an Image *does* have grips, surprised that this makes a difference...

**Comment by wizofaus** [comment]
It's certainly likely that this is better fix than the one I came up with, but does it still allow resizing via the bottom-right grip handle?

File: src/notation/internal/notationactioncontroller.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply src/notation/internal/notationactioncontroller.cpp:1247]: This code is needed for the "Edit element" item in the context menu. It automatically selects the default grip point of an element and starts "edit mode"

Comment (wizofaus) [reply src/notation/internal/notationactioncontroller.cpp:1247]: Ok but I believe if you use that mode then you *can't* use the mouse to move an image.  But that's the same as v3.

**Comment by RomanPudashkin** [comment]
> does it still allow resizing via the bottom-right grip handle?

Yes, sure :)

 >BTW: very clear commit messages, great to see! I hope this means we're no longer expected to always squash commits?
 
We need to squash only those commits that don't make much sense. For example, commits with code-style fixes, commits with some minor code-review fixes, etc. Small commits with detailed descriptions are still welcome

**Review by Eism — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #12587 #11900 on Win10, Mac12, LinuxMint 20.2 - **FIXED**

**Pull request merged**